From 423480ce3a34bba30119bc5ba101442274bc4662 Mon Sep 17 00:00:00 2001
From: Tom St Denis <tom.stdenis@amd.com>
Date: Fri, 4 Dec 2015 08:33:36 -0500
Subject: [PATCH 0813/1050] amdgpu/gfxv8:  Remove rogue break in
 gfx_v8_0_tiling_mode_table_init()

There was a rogue break in gfx_v8_0_tiling_mode_table_init() that broke TONGA
support.

Signed-off-by: Tom St Denis <tom.stdenis@amd.com>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/gfx_v8_0.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/gfx_v8_0.c b/drivers/gpu/drm/amd/amdgpu/gfx_v8_0.c
index 8be11b9..086cd55 100644
--- a/drivers/gpu/drm/amd/amdgpu/gfx_v8_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/gfx_v8_0.c
@@ -2207,7 +2207,6 @@ static void gfx_v8_0_tiling_mode_table_init(struct amdgpu_device *adev)
 				 BANK_HEIGHT(ADDR_SURF_BANK_HEIGHT_1) |
 				 MACRO_TILE_ASPECT(ADDR_SURF_MACRO_ASPECT_1) |
 				 NUM_BANKS(ADDR_SURF_8_BANK));
-				break;
 		mod2array[13] = (BANK_WIDTH(ADDR_SURF_BANK_WIDTH_1) |
 				 BANK_HEIGHT(ADDR_SURF_BANK_HEIGHT_1) |
 				 MACRO_TILE_ASPECT(ADDR_SURF_MACRO_ASPECT_1) |
-- 
1.9.1

